src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -739,6 +765,27 @@ private static boolean isDelimiter(final char ch, final char[] delimiters) {
         return false;
     }
 
+  //-----------------------------------------------------------------------
+    /**
+     * Is the codePoint a delimiter.
+     *
+     * @param codePoint the codePint to check
+     * @param delimiters  the delimiters
+     * @return true if it is a delimiter
+     */
+    private static boolean isDelimiter(final int codePoint, final char[] delimiters) {
```

Comment:
Thanks for pointing this, I created method first then generated it with eclipse by default it generated private so missed this one, I think this should be *public*.